sprints.md

```diff
@@ -8,8 +8,8 @@ startDate: 2018-07-16
 endDate: 2018-07-18
 startTime: "10:30"
 endTime: "21:30"
-venueName: "Semaphore Demo Room"
-location: "Room 417, Claude T. Bissell, 140 St. George Street"
+locationName: "Semaphore Demo Room"
+location: "Room 417, Claude T. Bissell building, 140 St. George Street"
```

Comment:
I think `building` is part of it's title—title case? `Claude T. Bissell Building`?

Reply:
👍